README.md

```diff
@@ -16,8 +26,18 @@ Be sure to install node modules:
 npm install
 ```
 
-It is no ready to start a local development environment:
+It is now ready to start a local development environment:
 
 ```shell
 npm start
 ```
+
+## Authors
```

Comment:
I don't feel like an Author, I feel more like a Creator and/or a Founder. I'll need to think about how I want this structured in the readme. I think for now, just cut this section. I'll come back to it.